docs/ansible/roles/debops.phpipam/introduction.rst

```diff
@@ -0,0 +1,28 @@
+Introduction
+============
+
+`phpIPAM`_  is an open-source web IP address management application (IPAM).
+Its goal is to provide light, modern and useful IP address management.
+It is php-based application with MySQL database backend, using jQuery
+libraries, ajax and HTML5/CSS3 features.
+
+
+The ``debops.phpipam`` role installs this ipam on a specified host with
+nginx as a webserver (using :ref:`debops.nginx`).
+
+.. _phpIPAM: https://phpipam.net/
+
+Installation
+~~~~~~~~~~~~
+
```

Comment:
The installation section can be dropped, since all of the DebOps roles are now installed as one multi-role repository.

Reply:
done